lib/std/meta.zig

```diff
@@ -277,10 +276,10 @@ test "std.meta.fields" {
 }
 
 pub fn fieldInfo(comptime T: type, comptime field_name: []const u8) switch (@typeInfo(T)) {
```

Comment:
You could change this to `@typeId`

Reply:
`@typeId` is deprecated in favor of `@typeInfo`